src/js/utils/__tests__/JSONUtil-test.js

```diff
@@ -0,0 +1,290 @@
+jest.dontMock('../JSONUtil');
+
+const JSONUtil = require('../JSONUtil');
+
+describe('JSONUtil', function () {
+
+  describe('#getObjectInformation', function () {
+
+    //
+    // Empty Object
```

Comment:
Let's remove these comments and organize tests with `describe` blocks for better output